# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/VideoUtils.h
# Commit: bcb45500ba22
# Full Hash: bcb45500ba2223a5e21bcc29922c1260d350694f
# Author: Paul Adenot <paul@paul.cx>
# Date: 2023-12-23 09:52:03
# Regressor Bug: 1749047
# File Overlap Count: 3
# Description:
#   Bug 1749047 - Wire up more codecs from VideoEncoder to the new FFmpeg PEM. r=chunmin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D196108
# ==============================================================================

diff -r a78bb8799ef4 -r bcb45500ba22 dom/media/VideoUtils.h
--- a/dom/media/VideoUtils.h	Fri Dec 22 21:39:12 2023 +0000
+++ b/dom/media/VideoUtils.h	Fri Dec 22 21:39:13 2023 +0000
@@ -259,7 +259,7 @@
 bool ExtractVPXCodecDetails(const nsAString& aCodec, uint8_t& aProfile,
                             uint8_t& aLevel, uint8_t& aBitDepth,
                             uint8_t& aChromaSubsampling,
-                            VideoColorSpace& aColorSpace);
+                            mozilla::VideoColorSpace& aColorSpace);
 
 // Extracts AV1 codecs parameter string.
 // See https://aomediacodec.github.io/av1-isobmff/#codecsparam
@@ -268,7 +268,7 @@
                             uint8_t& aLevel, uint8_t& aTier, uint8_t& aBitDepth,
                             bool& aMonochrome, bool& aSubsamplingX,
                             bool& aSubsamplingY, uint8_t& aChromaSamplePosition,
-                            VideoColorSpace& aColorSpace);
+                            mozilla::VideoColorSpace& aColorSpace);
 
 // Use a cryptographic quality PRNG to generate raw random bytes
 // and convert that to a base64 string.